README.md

```diff
@@ -27,6 +27,7 @@ npm install pino-colada
 - [merry](https://github.com/yoshuawuyts/merry)
 - [garnish](https://github.com/mattdesl/garnish)
 - [@studio/log](https://github.com/javascript-studio/studio-log)
+- [pino-http](https://github.com/pinojs/pino-http)
```

Comment:
yea that'd probably be too many; `pino-http` and `hapi-pino` are good additions tho 👌